modules/upload/upload.service.js

```diff
@@ -0,0 +1,119 @@
+require('dotenv').config();
+const azureStorage = require('azure-storage');
+const blobService = azureStorage.createBlobService(
+    process.env.STORAGE_ACCOUNT, 
+    process.env.ACCESS_KEY, 
+    process.env.AZURE_HOST
+);
+const containerName = 'images';
+const getStream = require('into-stream');
+const Jimp = require("jimp");
+const uniqid = require('uniqid');
+
+class UploadService {
+    async uploadFiles(files) {
+        return files.map(async file => {
+            const { createReadStream, filename } = await file;
+
+            const inputStream = createReadStream()
+            let fileBuffer;
+            const id = uniqid()
+
+            const inputBuffer = await new Promise((resolve, reject) => {
+                let chunks = [];
+                inputStream.once('error', (err) => {
+                    return reject(err);
+                });
+
+                inputStream.once('end', () => {
+                    fileBuffer = Buffer.concat(chunks);
+                    return resolve(fileBuffer);
+                });
+
+                inputStream.on('data', (chunk) => {
+                    chunks.push(chunk);
+                });
+            })
+
+            Jimp.read(inputBuffer).then(image => {
+                image.resize(1920, Jimp.AUTO).getBuffer(image.getMIME(), (err, buffer) => {
```

Comment:
Please, convert `resize` to `Promise` using `promisify` from the default node package `util`

Reply:
fixed